### PR TITLE
Add drop down menu

### PR DESCRIPTION
**Pull request by eminguyen**

### CODE REVIEW

File: src/components/NavDropdown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        </NavLink>
+        <NavLink activeClassName="selected" className="nav-item" to="/leaderboard">
+          <NavListItem text="Leaderboard" />
+        </NavLink>
+        <NavLink activeClassName="selected" className="nav-item" to="/profile">
+          <NavListItem text="Profile" />
+        </NavLink>
+      </section>
+
+      <Divider style={{ margin: '0.5rem 0' }} />
```
Comment (leestanley) [inline src/components/NavDropdown/index.jsx]: Not sure if this applies to styles, but AirBnB React styling says no padding in curly braces https://i.imgur.com/cZV1ZBS.png

Comment (leestanley) [reply src/components/NavDropdown/index.jsx]: https://github.com/airbnb/javascript/tree/master/react#alignment

File: src/components/NavDropdown/index.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        Sign Out
+      </div>
+    </nav>
+  );
+};
+
+NavDropdown.propTypes = {
+  onClick: PropTypes.func
+};
+
```
Comment (leestanley) [inline src/components/NavDropdown/index.jsx]: AirBnB ordering puts proptypes at the top https://i.imgur.com/Y1BYcrY.png - not sure if you wanted to follow this too

Comment (eminguyen) [reply src/components/NavDropdown/index.jsx]: I don't think we can do this with functional components

**Pull request merged**